### PR TITLE
assessment routes fix

### PR DESCRIPTION
**Pull request by aaron-muti-420**

### CODE REVIEW

File: app/Http/Controllers/System/AssessmentController.php [hunk 1]
```diff
@@ -14,7 +14,7 @@ class AssessmentController extends Controller
     public function index()
     {
         $assesment = assessment::all();
```
Comment (hpmouton) [inline app/Http/Controllers/System/AssessmentController.php:16]: Fix all variable names in controller, make sure spelling is correct.

**Review by hpmouton — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/System/AssessmentController.php (hunk 1 above)
Comment (hpmouton) [reply app/Http/Controllers/System/AssessmentController.php:16]: Also delete the storage folder in the root directory before pushing to resolve merging conflicts.

Comment (aaron-muti-420) [reply app/Http/Controllers/System/AssessmentController.php:16]: Fixed the above changes

**Review by hpmouton — APPROVED** [review]

**Comment by hpmouton** [comment]
Before we merge please solve your merge conflicts.

Step 1: From your project repository, check out a new branch and test the changes.

- git checkout -b aaron-muti-420-main main
- git pull https://github.com/aaron-muti-420/skillharbor-open.git main

Step 2: Merge the changes and update on GitHub.

- git checkout main
- git merge --no-ff aaron-muti-420-main
- git push origin main

**Pull request closed without merging**